packages/generator/package.json

```diff
@@ -5,6 +5,7 @@
   "license": "Apache-2.0",
   "scripts": {
     "start": "echo nothing to start",
+    "prepublish": "echo nothing to prepublish",
```

Comment:
I have added that so you can do 
`lerna exec yarn run prepublish`
this is the only package which have no prepublish script

Reply:
ok